### ISSUE TITLE
MP abnormal cytokinesis

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3214]
elongated multinucleate multiseptate vegetative cell, single septa between nuclei (FYPO:0001253) [single allele genotypes]

is a cytokinesis failure so should have abnormal cytokinesis parent\

**Comment by mah11** [comment]
FYPO:0001253 is a structure/morphology phenotype, and therefore can't have an is_a link to abnormal cytokinesis (a process phenotype).

There is a path to abnormal cytokinesis, but it goes via a has_part link and an output_of link. It's slightly easier to see in the graph for one of the is_a parents, FYPO:0000223:
![fypo_223](https://user-images.githubusercontent.com/5196259/35626706-6e5cf890-068e-11e8-8e37-9100c4dc4b59.png)

**Comment by ValWood** [comment]
so, based on that, shouldn't all multiseptate vegetative cell be retrieved by a query for cytokinesis"  (they aren't, but they are all cytokinesis defects)

I  guess we need to look at "out_put" of... ( I feel as though I have done this about 10 times, so maybe it works for some things and not for others...)

**Comment by mah11** [comment]
They do show up on the term page for abnormal cytokinesis. If the search doesn't find the same genes, that seems like a question for Kim about search behavior.

**Comment by ValWood** [comment]
just to check

I don't see
elongated multinucleate multiseptate vegetative cell, single septa between nuclei
on the abnormal cytokinesis page, but I do see


elongated multinucleate multiseptate vegetative cell, irregular septum position
inviable elongated multiseptate vegetative cell with fragmented DNA

which seem very similar?

**Comment by mah11** [comment]
Hmm. I looked specifically for FYPO:0000118 'multiseptate vegetative cell' when I last commented. It's odd, though, and definitely one for Kim to look at, because FYPO:0000133 'elongated multinucleate vegetative cell' appears on that page, and FYPO:0001253 is_a FYPO:0000133. I therefore don't understand why FYPO:0001253 wouldn't be on the FYPO:0000032 page if FYPO:0000133 is.

Closing here, because the ontology is OK.
